invokeai/frontend/web/src/features/system/components/SettingsModal/SettingsUpsellMenuItem.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Flex, Icon, MenuItem, Text, Tooltip } from '@invoke-ai/ui-library';
+import { useTranslation } from 'react-i18next';
+import type { IconType } from 'react-icons';
+import { PiArrowUpBold } from 'react-icons/pi';
+
+export const SettingsUpsellMenuItem = ({ menuText, menuIcon }: { menuText: string; menuIcon: IconType }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Tooltip label={t('upsell.professionalUpsell')} placement="right" zIndex={2}>
```

Comment:
Manually specifying zIndex can cause issues, especially on the workflows tab where `reactflow` does stuff to zIndex